.travis.yml

```diff
@@ -15,15 +15,12 @@ before_install:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
   - export CHROME_BIN=chromium-browser
-  # Upgrade NPM
-  - npm install -g npm
```

Comment:
why removing the upgrade NPM ?

Reply:
If we install node LTS we will have the according NPM version so for me that's not a useful step, it gains some seconds on the build :)
Did you have issues by not upgrading NPM ?

Reply:
no issue, just consistency for all other builds we have

Reply:
Well i can keep it :)